core/src/main/scala/org/apache/spark/status/LiveEntity.scala

```diff
@@ -341,7 +341,9 @@ private class LiveExecutorStageSummary(
       metrics.shuffleWriteMetrics.recordsWritten,
       metrics.memoryBytesSpilled,
       metrics.diskBytesSpilled,
-      isBlacklisted)
+      isBlacklisted,
+      null,
```

Comment:
Use named parameters when passing constant values to functions.

Reply:
Done